.drone.yml

```diff
@@ -3,143 +3,552 @@ kind: pipeline
 name: default
 type: kubernetes
 
-steps:
-- name: build_are
-  pull: if-not-exists
-  image: 340268328991.dkr.ecr.eu-west-2.amazonaws.com/acp/dind
-  commands:
-    - n=0; while [ "$n" -lt 60 ] && [ ! docker stats --no-stream ]; do n=$(( n + 1 )); sleep 1; done
-    - docker build -t app-$${DRONE_COMMIT_SHA} .
-  when:
-    branch: [ master, feature/* ]
-    event: push
+environment:
+  APP_NAME: appeal-rights-exhausted
+  PROD_ENV: are
+  STG_ENV: are-preprod
+  UAT_ENV: are-dev
+  BRANCH_ENV: are-branch
+  PRODUCTION_URL: www.appeal-rights-exhausted.homeoffice.gov.uk
+  IMAGE_URL: quay.io/ukhomeofficedigital
+  IMAGE_REPO: are-app
+  GIT_REPO: UKHomeOffice/AppealRightsExhausted
+  HOF_CONFIG: hof-services-config/Appeal_Rights_Exhausted
+  NON_PROD_AVAILABILITY: Mon-Fri 08:00-23:00 Europe/London
+  READY_FOR_TEST_DELAY: 20s
+  NOTIFY_STUB: stub
```

Comment:
Is ARE using notify?

Reply:
no, this is just here for when it does and so these files are consistent will the pipeline rollout elsewhere

Reply:
interesting because it's just really a calculator and it doesn't currently send emails.  I'm curious what you had in mind with emails
but fair enough if you have plans 🙂